core/src/main/resources/messages/messages_nl.properties

```diff
@@ -113,6 +113,9 @@ error.device.cannot.delete.last.one=Kan laatste apparaat niet verwijderen. Appar
 error.device.collection.empty=De geanonymiseerde gebruiker met ID ''{0}'' heeft geen apparaten. Dit is geen geldige toestand.
 error.device.not.found.index=De geanonymiseerde gebruiker met ID ''{0}'' heeft geen apparaat de index ''{1}''
 error.device.name.already.exists=Er bestaat al een apparaat met de naam ''{0}''
+error.device.app.version.not.supported=Yona app is te oud en moet worden ge�pdatet. De huidige versie is {2} maar de oudste ondersteunde versie voor {0} is {1} 
```

Comment:
geüpdatet -> geüpdate

Reply:
I feel the same, but alas, the spelling I used is correct. See https://nl.wiktionary.org/wiki/updaten/vervoeging